zipline/errors.py

```diff
@@ -191,6 +191,13 @@ class IncompatibleHistoryFrequency(ZiplineError):
 """.strip()
 
 
+class HistoryInInitialize(ZiplineError):
+    """
+    Raised when an algorithm calls history() in initialize.
+    """
+    msg = "history() can only be called in handle_data()"
```

Comment:
`history()` can also be called in `before_trading_start()`. I suggest changing this to either "history() *should* only be called in handle_data()", or "history() can not be called in initialize()"